benchmarks/bench_lasso.py

```diff
@@ -20,6 +20,7 @@
 
 def compute_bench(alpha, n_samples, n_features, precompute):
```

Comment:
Should we call this files  `bench_lasso_screening.py`. Sounds more explicit for future reference